src/pkg/projects/Microsoft.NETCore.DotNetHost/host.wxs

```diff
@@ -8,16 +8,22 @@
     <ComponentGroup Id="InstallSharedHostandDetectionKeys">
 
       <Component Id="cmpCoreHost" Directory="DOTNETHOME" Guid="{45399BBB-DDA5-4386-A2E9-618FB3C54A18}" >
-        <File Id="fileCoreHostExe" KeyPath="yes" Source="$(var.HostSrc)\dotnet.exe" />
+        <File Id="fileCoreHostExe" KeyPath="yes" Source="$(var.HostSrc)\dotnet.exe">
+          <CopyFile Id="copyFileCoreHostExe" DestinationProperty="$(var.Program_Files)dotnet" />
```

Comment:
I assume the system is smart enough to handle the common case where the two destinations are the same?

I don't feel qualified to review this code, but I approve of the change.

Reply:
I have testing this case: 'the two destinations are the same' and it is handled by the installer gracefully and without error.

Reply:
yes, when the targets for FileCopy are the same, it only executes once per the documentation:
_If the specified destination directory is the same as the directory containing the original file and the name for the proposed source file is the same as the original, then no action takes place._